### DISCUSSION TITLE
Docs Update Project Phase 5: Maintain ESLint

### DISCUSSION THREAD

**Discussion by bpmutter** [eslint/eslint#16905] [Ideas]
This discussion is the outline for phase 5 of the [Documentation Update Project](https://github.com/eslint/eslint/issues/16365) for updating the Maintain ESLint documentation.

## Overview

I don't think there's much that has to be done in this phase. I'm not one of the maintainers, but the docs seem generally well written to me. I don't see much copy editing to do. 

One thing that I think could be interesting, but may lie outside the scope of this project, is to include some more information about how the ESLint organization is structured. 

## Tasks

- Copy edits on all the maintain ESLint docs. I think I'd do this in one PR because the changes will be fairly cosmetic since the docs are well written and clear.
- Any other tasks that the core team would like
- (potentially) new 'How ESLint is Maintained' page
  - overview of different roles at ESLint
    - link to the [Team](https://eslint.org/team/) page
    - maybe additional explanation of roles?
  - any other info about the org structure.
    - for example, how ESLint fits in with the [OpenJS Foundation](https://openjsf.org/)
  - How ESLint is funded
    - breakdown of the different funding sources with links to those: open collective, github sponsors, tidelift (taken from [this blog post](https://eslint.org/blog/2023/01/eslint-2022-year-review/#income-review))
  - info about how readers can become part of maintainer team 
  - **note**: i think this page would be a nice addition to the docs to be more 'open' about how ESLint works. in addition to having interesting and useful info for folks who want to learn more about ESLint, I think this page could help show other open-source projects looking to emulate ESLint how it's structured and funded.

**Comment by nzakas** [comment]
I like the idea for a "How ESLint is maintained" page. I think that could really help when we are talking to potential sponsors to have them better understand what goes into maintaining the project. 

Similarly, "How ESLint is funded" also seems like a good idea.

So overall, totally on board. 👍
